lib/modules/showImages.js

```diff
@@ -1352,7 +1350,46 @@ function generateIframe(options: IframeMedia) {
 	element.independent = true;
 
 	keepMediaVisible(iframeNode);
```

Comment:
One last thing: This should be `keepMediaVisible(iframeFooter)` so that the footer follows the iframe when it is moved towards the left.